### DISCUSSION TITLE
Scanning a QR Code with UnityNativeCamera in Android

### DISCUSSION THREAD

**Discussion by amenezescheil** [yasirkula/UnityNativeCamera#100] [Q&A] [answered]
Hello all, 

There are 2 crucial features that I need to function at the same time in my Unity Android App:
- Use the android camera's zoom feature (works!)
**- Use the android camera's QR code scanner (doesn't work)**

Any idea how to enable QR code scanning in this plugin?
Thanks!

**Comment by yasirkula** [comment, marked as answer]
This plugin doesn't have advanced capabilities. I think this would require writing the camera API from scratch whereas this plugin just launches the device's native camera application and waits for it to return an image/video path.

**Reply by amenezescheil** [reply]
Thank you!

Edit: Upon doing a little research, I've realized that I mistakenly assumed that the "native camera app" was Samsung's camera app on my phone. The ACTUAL native camera app is the default Android one, not Samsung's. The Native Camera App (android) doesn't have QR code scanning capabilities, and these kinds of functionalities are typically programmed by the company (e.g. Samsung). Just thought I'd add that little tidbit of information I discovered, please correct me if I'm wrong!

**Reply by yasirkula** [reply]
Thank you for the additional information! I myself didn't notice that there are 2 different stock camera apps in Samsungs.
